README.md

```diff
@@ -1,17 +1,22 @@
-## Sensu-Plugins-skel
+# Sensu Plugin Skeleton
 
-[![Build Status](https://travis-ci.org/sensu-plugins/sensu-plugins-skel.svg?branch=master)](https://travis-ci.org/sensu-plugins/sensu-plugins-skel)
-[![Gem Version](https://badge.fury.io/rb/sensu-plugins-skel.svg)](http://badge.fury.io/rb/sensu-plugins-skel)
-[![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-skel.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-skel)
+All the files you'll need to start your own plugin written in Ruby [for Sensu](https://github.com/sensu/sensu), the monitoring framework.
 
-## Functionality
+## Useful to Me?
 
-## Files
+* Copy the files in this repository if you want to write a Ruby plugin following best practices of layout and structure
+* To write the actual functionality, you'll need to import [Sensu Plugin gem](https://github.com/sensu-plugins/sensu-plugin)
+* If you'd prefer writing in Python, use the [Python Plugin library](https://github.com/sensu-plugins/sensu-plugin-python)
```

Comment:
For the moment I still encourage copying and packaging it as a gem an addition to a pypi packages. The reason is that this now introduces 2 ways to install plugins. This is especially true since many plugins that have python, ruby, (bash|powershell) in the same project. While this is a larger discussion: https://github.com/sensu-plugins/community/issues/70 I just wanted to point out and decide if we want to adjust any verbiage here.

Reply:
Ok I think I can mention that as another checkbox below 9: "Regardless of the programming language, complete the gem packaging so its easily installed." How's that read? Any link on further details?

Reply:
Not quite yet a link but that being said here is a good example of why:
![image](https://user-images.githubusercontent.com/3145127/32974269-b68b01ba-cbb1-11e7-9bbc-3de14fbb41c2.png)
This is the mongo plugin if you are wondering.

Reply:
This stance will likely change at some point in the future as sensu will no longer NEED a ruby runtime so it's packaging may no longer be the defacto standard. Once this happens it opens up a lot of requests regarding reducing footprint for special use cases such as embedded devices and containers.

Reply:
@majormoses I'm with you. It's a great call to mention is and I'm happy linking to Mongo as an example. Doesn't have to be canonical, just something.